pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -160,18 +155,19 @@ func IsGreaterOrEqualToBaseVersion(version string, baseVersion string) (bool, er
 	return false, nil
 }
 
-func getGithubReleases() ([]releaseTags, error) {
-	githubReleaseUrl := "https://api.github.com/repos/openshift/origin/releases"
-	resp, err := getResponseBody(githubReleaseUrl)
+func getGithubReleases() ([]string, error) {
+	var releaseTags []string
+	client := github.Client()
+	listOptions := &goGithub.ListOptions{}
```

Comment:
It would be good to have github logic directly in the package we maintain. 

following can be added to `pkg/util/github/github.com`
```golang
func ListOptions() *github.ListOptions {
	return &github.ListOptions{}
}

func IsRateLimitError(err error) bool {
	if _, ok := err.(*github.RateLimitError); ok {
		return true
	}
	return false
}
```

Here it can consumed as 
``` golang
listOptions := github.ListOptions()
	releases, _, err := client.Repositories.ListReleases("openshift", "origin", listOptions)
	if err != nil {
		if github.IsRateLimitError(err) {
			return nil, fmt.Errorf("Hit github rate limit: %v", err)
		}
		return nil, err
	}
```

Reply:
Okay.. will add this to our package